### ISSUE TITLE
IRIS Q&As Content Review: Batch 2, Life Insurance

### ISSUE THREAD

**Issue by lwalsh617** [department-of-veterans-affairs/va.gov-team#14241]
Attached copy deck to be reviewed/edited by DEPO and then by SMEs. 

- [ ] DEPO to review/edit copy (BETH)

- [ ] DEPO to route copy to SMEs for approval (BETH)

- [ ] PW to create new GH ticket for Drupal build of pages and attach final copy deck (RANDI)

[lc-mvp-lifeinsurance-faqs-copy-v0.1-100120-RH-edit-LW.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5311834/lc-mvp-lifeinsurance-faqs-copy-v0.1-100120-RH-edit-LW.docx)

 ###Acceptance Criteria

- [ ] Final copy deck ready for CMS

**Comment by lwalsh617** [comment]
@MickinSahni @brianalloyd -- I put a needs-grooming label on this because milestone and estimate still need to be assigned and requires Beth's input.

**Comment by laurwill** [comment]
@bethpottsVADEPO and @RLHecht , I think we can close this out — this batch of IRIS Q&As contains 3 items.
- What is TSGLI and am i eligible — covered on TSGLI page
- What is VGLI and am I eligible — covered on VGLI page
- Do all VA life insurance policies pay dividends? — covered on [this R&S page](https://www.va.gov/resources/life-insurance-dividend-payment-options/)

Is there a special requirement to migrate IRIS Q&As to single R&S Q&As? If not, it looks like we have all this info accounted for.

**Comment by bethpottsVADEPO** [comment]
I agree! Let's close it out. As long as the content is covered somewhere, we're good.

**Comment by mistymg** [comment]
Moved to closed but not actually showing up as closed on board.
